specs/v1/field.schema.json

```diff
@@ -51,6 +51,10 @@
                 "is_null": {
                     "description": "Can the field have null values",
                     "type": "boolean"
+                },
```

Comment:
A field that should be hidden is a rendering property and should not necessarily be inside the attributes block. 

it's handled by field.render.input = hidden